### ISSUE TITLE
Applying a common, uncommon or rare ornament to  legendary armor

### ISSUE THREAD

**Issue by mannyrye** [DestinyItemManager/DIM#8579]
### DIM Version

Version 7.25.0 (release), built on 7/10/2022, 5:13:31 PM

### Browser Details

All browsers

### OS Details

At least iOS, MacOS and Edge on Xbox

### Describe the bug

When applying an ornament made from common, uncommon or rare armor to a legendary piece of armor it makes it look like it's not legendary on DIM (it appears correctly on Destiny). 
![DBE4C901-7AA6-48C3-ADC4-731727B40DBD](https://user-images.githubusercontent.com/20385998/179159648-c30fac5b-ce08-4d21-88e9-09c21954e32a.jpeg)
![D4720348-5C89-40F3-BE1A-7D2D0A03270D](https://user-images.githubusercontent.com/20385998/179159651-5265673e-5535-4c63-8765-24ec80dd2839.jpeg)
![9A7D4D74-A3B4-4C4C-BBD7-E7290C1B19A6](https://user-images.githubusercontent.com/20385998/179159653-6818e290-8a0a-45f4-8a5d-7cbb2eff4df4.jpeg)
![C4DBAA93-EB17-4C8B-86D1-4AD8C1001315](https://user-images.githubusercontent.com/20385998/179159655-1520bc18-98f6-4c9f-8259-fd749a2e51e1.jpeg)

**Comment by robojumper** [comment]
I don't think this can be addressed on the DIM side -- when the API generates the icons from game data it adds the blue background to the icons so that's all we have.

**Comment by delphiactual** [comment]
What does the item look like in the Destiny Companion App?

**Comment by mannyrye** [comment]
The companion app does the same as DIM 😔 ![image](https://user-images.githubusercontent.com/20385998/179262546-13c998c5-cead-49b0-ba20-fef968e71d65.jpeg)

**Comment by bhollis** [comment]
We've asked the Bungie API team to change how they compose item icons to not include the background, but we have no expectation that it'll ever happen.
